Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml

```diff
@@ -22,6 +22,10 @@
     tags:
     - CaptainSabre
   - type: DisarmMalus
+  - type: Tool
```

Comment:
sabre can pry doors wtf

Reply:
QoL improvement so captain doesn't have to seek out a crowbar all of the time. It functions similarly to the fireaxe's prying ability. It's speed has been lowered significantly. It is still better to have a crowbar, but the sword will work if there is no other options.